06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndex.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * inverted index class
+ * @class
+**/
+class InvertedIndex {
+  /**
+  * class constructor
+  * @constructor
+  **/
+  constructor() {
+    this.fileIndices = {};
+  }
+  /**
+   * Set Index - Sets the indices of all indexed files
+   * @param {String} filename - Name of the indexed file
```

Comment:
For consistency sake on your code, shouldn't this parameter name be in camelCase?